### DISCUSSION TITLE
单页应用支持

### DISCUSSION THREAD

**Discussion by jalenzz** [walinejs/waline#291] [Q&A] [answered]
可否提供一个单页应用使用 Waline 的代码样例，谢谢

**Comment by lizheming** [comment]
Hopes 君最近整理的 1.0 版本有针对 SPA 的适配，可以请 @Mister-Hope 帮忙提供一下示例。

**Comment by Mister-Hope** [comment, marked as answer]
文档就是一个样例。。。。。详见 : https://github.com/walinejs/waline/blob/main/docs/.vuepress/theme/components/Waline.ts

而且目前文档已经添加了详细的spa介绍。 https://waline.js.org/guide/client/spa.html
